docs/bugs_by_version.json

```diff
@@ -1,565 +1,565 @@
 {
     "0.1.0": {
         "bugs": [
-            "ZeroFunctionSelector", 
```

Comment:
This is a generated file.

Reply:
I updated the generating script now in the second commit.